assets/queries/ansible/aws/s3_bucket_without_encryption/query.rego

```diff
@@ -0,0 +1,41 @@
+package Cx
+
+CxPolicy [ result ] {
+  	document := input.document[i]
+  	task := getTasks(document)[t]
+    s3_bucket := task["amazon.aws.s3_bucket"]
+
+    not s3_bucket.encryption
+
+	result := {
+                "documentId": 		document.id,
+                "searchKey":        sprintf("name={{%s}}.{{amazon.aws.s3_bucket}}", [task.name]),
+                "issueType":		"MissingAttribute",
+                "keyExpectedValue": "aws_s3_bucket.encryption is defined",
+                "keyActualValue": 	"aws_s3_bucket.encryption is undefined",
+              }
+}
```

Comment:
I think the encryption is disabled only if you set it to "none" check the link bellow.

"Describes the default server-side encryption to apply to new objects in the bucket. **In order to remove the server-side encryption, the encryption needs to be set to 'none' explicitly**." Source:
https://docs.ansible.com/ansible/latest/collections/amazon/aws/s3_bucket_module.html#parameter-encryption